app/models/user.rb

```diff
@@ -494,6 +494,19 @@ def update_purchasing_power_parity_excluded_products!(external_ids)
     end
   end
 
+  def update_reply_to_emails!(reply_to_emails_data)
+    products.where.not(reply_to_email: nil).update_all(reply_to_email: nil)
+
+    reply_to_emails_data&.each do |email_data|
+      email = email_data[:email]&.strip
+      next if email.blank?
```

Comment:
This may be confusing UX I would imagine if I am trying to remove an email. I may just make it blank and expect it to go away.